pkg/api/index.go

```diff
@@ -50,8 +50,8 @@ func (hs *HTTPServer) setIndexViewData(c *contextmodel.ReqContext) (*dtos.IndexV
 	// Locale is used for some number and date/time formatting, whereas language is used just for
 	// translating words in the interface
 	acceptLangHeader := c.Req.Header.Get("Accept-Language")
-	locale := "en-US"
-	language := "" // frontend will set the default language
+	locale := "en-US" // frontend will set the default locale, but we set English as default
```

Comment:
I think this comment is a bit confusing - where is the default being set?

I think it could be removed, or reworded to like `default to en-US formatting, but use the accept-lang header or user's preference`